Mathlib/Algebra/ContinuedFractions/Computation/Approximations.lean

```diff
@@ -230,16 +231,14 @@ theorem fib_le_of_contsAux_b :
         -- use the IH to get the inequalities for `pconts` and `ppconts`
         have ppred_nth_fib_le_ppconts_B : (fib n : K) ≤ ppconts.b :=
           IH n (lt_trans (Nat.lt.base n) <| Nat.lt.base <| n + 1) (Or.inr not_terminatedAt_ppred_n)
-        suffices (fib (n + 1) : K) ≤ gp.b * pconts.b by
-          solve_by_elim [_root_.add_le_add ppred_nth_fib_le_ppconts_B]
+        gcongr
```

Comment:
IMO changes like this one aren't necessarily improvements: if new `gcongr` lemmas are added then this proof could break in a tricky way, whereas the old `suffices` block avoided this. I know we don't strictly have a rule against non-terminal `gcongr`, but it is a flexible tactic (and in this particular case, later followed by a rigid tactic). Here, my preference would be to keep the `suffices` block with its proof replaced by `gcongr`, but there's also the option of giving the pattern to `gcongr`

Reply:
Reverting to `suffices ... by gcongr`.